.github/workflows/mega-linter-for-runner.yml

```diff
@@ -113,61 +258,91 @@ jobs:
             megalinter-reports
             mega-linter.log
 
-      # Set APPLY_FIXES_IF var for use in future steps
```

Comment:
Please freshen this PR from main, so we don't undo https://github.com/oxsecurity/megalinter/pull/2957 's changes.